changes/1557-wangpeibao.md

```diff
@@ -0,0 +1,2 @@
+change schema.py/field_class_to_schema
+add frozenset type to the tuple
```

Comment:
```suggestion
change `schema.field_class_to_schema` to support `frozenset` in schema.
```

Reply:
Sorry, I do not know how to add tests.

Reply:
you need to commit this suggested change.